db/primary_migrate/20231024170146_create_email_data_table.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEmailDataTable < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension "citext"
```

Comment:
Am I correct in assuming we won't need anything additional to be installed in deployed environments for this to be enabled?